creativecloud/features/firefly/firefly-masonry.js

```diff
@@ -87,7 +87,8 @@ function processMasonryMedia(gridDiv, allP, mediaDetail) {
   handleTouchDevice(mediaContainer, 2000);
 
   const spanWidth = mediaDetail.spans[lastIndex];
-  mediaContainer.classList.add(`ff-grid-${spanWidth.trim().replace(' ', '-')}`);
+  const spanClass = spanWidth === undefined ? 'span-4' : `${spanWidth?.trim().replace(' ', '-')}`;
```

Comment:
```suggestion
  const spanClass = spanWidth ? 'span-4' : `${spanWidth?.trim().replace(' ', '-')}`;
```

Reply:
Thanks for the review @aishwaryamathuria. I believe above will set the span-4 for all images even when it's defined in word doc.
I can use the below code. please let me know if you see any concern with that.
`const spanClass = spanWidth ? `${spanWidth?.trim().replace(' ', '-')}` : 'span-4';`